Romo/lib/Drive/RomoAV/RAVVideoOutput.m

```diff
@@ -48,56 +50,21 @@ - (id)init
 {
     self = [super init];
     if (self) {
-
-#ifndef SIMULATOR
-        av_log_set_level(AV_LOG_QUIET);
-        
-        _sourceFrame      = avcodec_alloc_frame();
-        _destinationFrame = avcodec_alloc_frame();
-        _outputBuffer = NULL;
-        
-        _frameDecodeQueue = dispatch_queue_create("Frame Decode Queue", DISPATCH_QUEUE_SERIAL);
-        _frameRenderQueue = dispatch_queue_create("Frame Render Queue", DISPATCH_QUEUE_SERIAL);
-        
-        _isInitialized = NO;
-        
-        avcodec_register_all();
-        _codec = avcodec_find_decoder(CODEC_ID_H264);
-        
-        _codecCtx = avcodec_alloc_context3(_codec);
-        
-        // Codec settings:
-        _codecCtx->width = 0;
-        _codecCtx->height = 0;
-        _codecCtx->flags2 |= CODEC_FLAG2_FAST;
-        _codecCtx->pix_fmt = PIX_FMT_YUV420P;
-        
-        avcodec_open2(_codecCtx, _codec, NULL);
         
         CGRect frame = [UIScreen mainScreen].bounds;
         
-        CGFloat videoAspectRatio = 480.0 / 640.0;
-        CGFloat frameAspectRatio = frame.size.width / frame.size.height;
-        
-        CGFloat k = videoAspectRatio / frameAspectRatio;
-        frame.size.width *= k;
-        
         self.peerView = [self prepareOutputViewWithFrame:frame];
-        self.peerView.transform = CGAffineTransformMakeScale(-1, 1);
         
-        _isVideoFrameQueueActive = YES;
-#endif
-        
-        [[NSNotificationCenter defaultCenter] addObserver:self
-                                                 selector:@selector(applicationWillResignActive:)
-                                                     name:UIApplicationWillResignActiveNotification
-                                                   object:nil];
+//        [[NSNotificationCenter defaultCenter] addObserver:self
```

Comment:
Why do you skip adding the observer? You also seem to be removing it without ever adding it.

Reply:
These two observers notifies the view to pause/resume renderer. But we are no longer using libav renderer.